test-skripsi/test-1.py

```diff
@@ -0,0 +1,41 @@
+
+def calculate_discount(price, discount):
+    discounted_price = price * discount
+    return discounted_price
+
+def factorial(n):
```

Comment:
The `factorial` function lacks a base case for `n == 0`. According to the mathematical definition of factorial, `0!` is `1`. Consider adding a condition to handle this case explicitly.